Feat:Want to implement Backend of Contact page #555

nishant0708 opened the pull request:
To make it work
1) cd backend
2)replace your details
![image](https://github.com/Anushkabh/krishiconnect/assets/101548649/6830f2f3-11d0-4f0c-bf98-7f8a426c5669)
here
3)node server.js



https://github.com/Anushkabh/krishiconnect/assets/101548649/1c1f656c-bb9a-4005-8612-d10f979cc376

himanshu-03 commented:
Hey @nishant0708 !

Please resolve the merge conflicts. Our repo underwent with a series of major changes since last 2 weeks. I request you to please sync the latest changes and resolve the conficts raised.

Make sure the files are in the desired directories

nishant0708 commented:
Okay I will

nishant0708 commented:
hey @himanshu-03 check this pr now i have updated this and i have updated New Templates as well for mail its much better than you can see them in video 
![Screenshot 2024-06-29 184002](https://github.com/Anushkabh/krishiconnect/assets/101548649/6e5b8de9-7821-4dd2-8926-0f3c955ecd2e)
![Screenshot 2024-06-29 184023](https://github.com/Anushkabh/krishiconnect/assets/101548649/80921d37-7abc-49e8-a564-c82b499af9ae)

Can you please please Assign it level 3

himanshu-03 commented:
Hello @nishant0708 !!

To make it work we need to put our email and password in it?

nishant0708 commented:
> Hello @nishant0708 !!
> 
> To make it work we need to put our email and password in it?

Your email and app-password

himanshu-03 commented:
> > Hello @nishant0708 !!
> > To make it work we need to put our email and password in it? Hey @himanshu-03
> 
> Your email and app-password
> 
> To generate it you have to go to Gmail-> manage your account -> enable 2step verification ->click on 2 step verification and then you will see app password ![IMG_20240630_131514572](https://private-user-images.githubusercontent.com/101548649/344450657-d31ad875-4574-4a4c-9ad2-d8ef3e386878.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk3MzQxMjMsIm5iZiI6MTcxOTczMzgyMywicGF0aCI6Ii8xMDE1NDg2NDkvMzQ0NDUwNjU3LWQzMWFkODc1LTQ1NzQtNGE0Yy05YWQyLWQ4ZWYzZTM4Njg3OC5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYzMFQwNzUwMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ZTA3YjY5ODFkZGVjY2FhMTczZDc3MGU2NDNhN2VlZmE2ZDYxMzJhZmI1MTAyMzVlMjBmZDZkY2MwMzhjNDhkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.ExYIVW642GGIWyLC2MRTO5fHPfOZr7A_TtFWwaTGZJg)
> 
> Generate password here and save it somewhere
> 
> This will be the password which you need

@Anushkabh please make a note of this and do the necessary changes by adding you pass and email in the secret variables in repo and vercel configs.

nishant0708 commented:
hey @himanshu-03 if everything works fine can you assign it to me

himanshu-03 commented:
Hey @nishant0708 !

Resolve this conflicts too

nishant0708 commented:
Just wait doing it too now

nishant0708 commented:
Hey @himanshu-03 please check it now

himanshu-03 commented:
> Hey @himanshu-03 please check it now. Can it be considered as of level 3 as I have also email templates . And also almost made it twice

Great work there @nishant0708 !

Sadly, it will be counted as a level 2 contribution only.

nishant0708 commented:
hey @himanshu-03 no issues just can you assign this pr to me as well as  can i start working on issue #612  now
